docs/DeveloperGuide.md

```diff
@@ -260,7 +259,192 @@ Note that only the `Category` field is allowed to be specified multiple times, w
 a `ParserException` is thrown. Another noteworthy point is that `Category` that are added are to be case-insensitively unique and can only be up to 
 a specified value in the `UniqueCategoryList` class. Else, a `ParserException` would be thrown.
 
+#### Get Total Expenditure
+
+##### Overview
+
+The `get_total_expenditure` command returns the total expenditure across a given month among all `expense` transactions in UniCa$h, with an optional filter for a given category.
+
+The activity diagram of getting the total expenditure is as shown below
+
+<img src="images/unicash/GetTotalExpenditureActivityDiagram.png" width="1026" />
+
+The following sequence diagram shows how the different components of UniCash interact with each other
+
+<img src="images/unicash/GetTotalExpenditureSequenceDiagram.png" width="1955" />
```

Comment:
I can't seem to view markdown preview, so I'm not too sure if the sizing for this is correct, but just in case it isn't, do remember to ensure that the diagram text of a similar font size to the surrounding text

Reply:
This is the default size of the image, will push a patch PR if the image scaling is off